data/human/ships.txt

```diff
@@ -1260,6 +1260,63 @@ ship "Cruiser"
 
 
 
+ship "Cutthroat"
+	sprite "ship/pirate cutthroat"
+	thumbnail "thumbnail/pirate cutthroat"
+	attributes
+		category "Light Warship"
+		"cost" 1100000
+		"shields" 2800
+		"hull" 600
+		"required crew" 4
+		"bunks" 6
+		"mass" 171
+		"drag" 3
+		"heat dissipation" .8
+		"fuel capacity" 500
+		"cargo space" 27
+		"outfit space" 282
```

Comment:
So, it loses 43 cargo to gain 22 outfit space? That's actually fairly reasonable.